includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -1861,4 +1865,29 @@ public function query( $query_vars ) {
 
 		return $products;
 	}
+
+	/**
+	 * Get data to save to a lookup table.
+	 *
+	 * @since 3.6.0
+	 * @param int    $id ID of object to update.
+	 * @param string $table Lookup table name.
+	 * @return array
+	 */
+	protected function get_data_for_lookup_table( $id, $table ) {
+		if ( 'wc_product_meta_lookup' === $table ) {
+			$price_meta   = (array) get_post_meta( $id, '_price', false );
+			$manage_stock = get_post_meta( $id, '_manage_stock', true );
+			$stock        = 'yes' === $manage_stock ? wc_stock_amount( get_post_meta( $id, '_manage_stock', true ) ) : null;
```

Comment:
_manage_stock should probably be _stock here, I tested and it did not update the stock status in the lookup table, it always stays same, think it is because of the wrong meta key being used here.